webroot/results/admin/export_public.php

```diff
@@ -157,6 +154,10 @@ function exportPublic ( $sources ) {
         echo '.';  # shows both Apache and the user that the script is doing stuff and not hanging
       }
     }
+    //Check if any errors while fetching data
+    if (($error = mysql_error()) !== '') {
```

Comment:
This script is only run by admins, so I think it's okay to dump sensitive information here. I'd be more worried about getting emailing working properly from php, lol.